### DISCUSSION TITLE
Hide NPC name

### DISCUSSION THREAD

**Discussion by Th7bo** [Minestom/Minestom#1868] [Help]
I've been trying to remove/hide the display name of an NPC. I am currently trying to create a team, set its NameTagVisibility tag to NEVER and adding the npc's to that team. It is adding the NPC's to the team but the nametag is still visible: 
![image](https://github.com/Minestom/Minestom/assets/60064927/4a248da3-ece7-4936-bb64-32f2166632c7)
![image](https://github.com/Minestom/Minestom/assets/60064927/6ccfb519-7a61-4e96-bcbd-34e33533e907)
![image](https://github.com/Minestom/Minestom/assets/60064927/64744686-f9cc-416e-a31a-5f6fe3920d3f)
![image](https://github.com/Minestom/Minestom/assets/60064927/e3291fde-b20b-4611-8c9f-ded9051e191f)
![image](https://github.com/Minestom/Minestom/assets/60064927/7aafb2b0-251a-426d-8ceb-54c2665c7f20)

**Comment by Obyvante** [comment]
Are you sure the players are also on the same team as NPCs?

**Reply by Th7bo** [reply]
I'm unsure, lemme see and try

**Reply by Th7bo** [reply]
No that didn't seem to work, I've tried `player.setTeam(NPC.team);` and `NPC.team.addMember(player.getUuid().toString());`
![image](https://github.com/Minestom/Minestom/assets/60064927/a50810f9-3a13-4e7f-9cee-f68eda60ffac)
